sync/fromDisk/assessments.sql

```diff
@@ -71,16 +71,34 @@ WHERE NOT EXISTS (
         AND a.deleted_at IS NULL
 );
 
+-- BLOCK insert_fake_ps_exam_if_needed
+WITH course_result AS (
+    INSERT INTO courses
+        (course_id, rubric)
+    VALUES
+        (1, 'CBTF')
+    ON CONFLICT (course_id) DO NOTHING
+)
+INSERT INTO exams
```

Comment:
We probably want a safer check than this in production? Someone putting `"examID": 999` in their config would actually make it in the prod DB and make that ID not available?

Reply:
Good point. We can run this code when in dev mode, but in prod we shouldn't do this at all.

I guess in prod we should just fail to sync if someone has an invalid examId?

Reply:
Ah, now I see what I did. The code above is only designed for dev mode. In prod we will set `config.select_exams_by_id = true`, so that we will never insert anything into the `exams` table, but rather we'll just check that the exam already exists.